### DISCUSSION TITLE
CVAT for geospatial

### DISCUSSION THREAD

**Discussion by AchimGaedkeLynker** [cvat-ai/cvat#8705] [General]
I started using CVAT for annotating aerial imagery - as alternative to [QGIS](https://www.qgis.org/).
I love the CVAT interface and the fact that it is an easily accessible browser application.

There are three observations I'd like to share and see someone has an opinion/solution:

* CVAT strips all metadata from images, esp the georeferencing
* CVAT is limited to 3 channels or one image/representation per annotation frame (e.g. I can't switch to a view highlighting water...)
* The polygon tools in QGIS are more flexible (mutliple rings/holes possible, flexible set operations) - that comes at a complexity cost though...

For geospatial data the first point is a "make or break" issue, also mentioned at https://github.com/cvat-ai/cvat/issues/6595

The location/extent of the image are stored either as metadata in the image or as a separate "world file".
Both are not recoverable when "simply" importing a collection of tiles of aerial imagery.

I found 3 ways of dealing with it:

* Use shared storage: https://docs.cvat.ai/docs/administration/basics/installation/#share-path - so you can refer to the original file by the frame name and retrieve the necessary metadata (and original resolution/added bands)
* Use image tags to store the coordinate reference system and raster transformation
* Use an external database outside CVAT for those data

I use (at the moment) the first way to work around it...

Does anyone have another solution or a plugin I could use (or modify) for geospatial data?

**Comment by joaomedeirosr** [comment]
Hi, @AchimGaedkeLynker, i have this same problem, can you find any way to solve this ?
